### PR TITLE
Change mandatory param wayPointType in GetWayPoints

### PR DESCRIPTION
**Pull request by OHerasym**
### PR Changes:
- In RPC GetWayPoints param wayPointsType mandatory is **true**

Related: [APPLINK-24153](https://adc.luxoft.com/jira/browse/APPLINK-24153)

### CODE REVIEW

**Comment by OHerasym** [comment]
@anosach-luxoft , @Kozoriz , @LuxoftAKutsan , @dtrunov , @VVeremjova , Please review

**Comment by LuxoftAKutsan** [comment]
@OHerasym probably same commit should be in CustomSDL?

**Comment by OHerasym** [comment]
@LuxoftAKutsan , There is only GENIVI label

**Comment by Kozoriz** [comment]
Reviewed

**Comment by LuxoftAKutsan** [comment]
Reviewed

File: src/components/interfaces/HMI_API.xml [hunk 1]
```diff
@@ -3407,7 +3407,7 @@
   </function>
   <function name="GetWayPoints" functionID="GetWayPointsID" messagetype="request">
     <description>Request for getting waypoint/destination data.</description>
-    <param name="wayPointType" type="Common.WayPointType" defvalue="ALL" mandatory="false">
+    <param name="wayPointType" type="Common.WayPointType" defvalue="ALL" mandatory="true">
       <description>To request for either the destination only or for all waypoints including destination</description>
     </param>
```
Comment (dtrunov) [inline src/components/interfaces/HMI_API.xml]: You need to change version according with https://adc.luxoft.com/confluence/display/APPLINK/API+xml+versioning+approach 
and please add business analyst

**Comment by dtrunov** [comment]
Reviewed

**Comment by OHerasym** [comment]
@CallousOdesa , Please review

**Comment by TMelnyk** [comment]
reviewed

**Comment by anosach-luxoft** [comment]
Reviewed

File: src/components/interfaces/HMI_API.xml (hunk 1 above)
Comment (OHerasym) [inline src/components/interfaces/HMI_API.xml]: @dtrunov , Fixed, Added business analyst to review

**Comment by OHerasym** [comment]
Build fails on travis, because of travis issue by timeout

**Comment by AGritsevich** [comment]
@Anatoliy-Leshin, this pull request ready to merge but wait until finished QtBuild. Don't calc it to metrics.

**Pull request merged**